api/nakadi-event-bus-api.yaml

```diff
@@ -1648,6 +1698,25 @@ definitions:
         description: |
           Offset of the event being pointed to.
 
+  ShiftedCursor:
+    allOf:
+      - $ref: "#/definitions/Cursor"
+      - type: object
+        properties:
+          shift:
```

Comment:
json schema defines `number` as 
> An arbitrary-precision, base-10 decimal number value, from the JSON "number" production

so this should not be limited to 2^32 or even 2^64. 

in case there is any limit from nakadi side, it is worth pointing out of course.